README.md

```diff
@@ -24,6 +29,14 @@ Once you have saved the new theme, go to `Home -> COmanage -> CO Settings` and s
 You can replace the logo by uploading an image with name `logo.png` to `<path-to-comanage-registry>/local/webroot/img/` directory.  
 You can also replace the favicon by uploading one with name `favicon.ico` to `<path-to-comanage-registry>/webroot/` directory.
 
+## Compatibility matrix
+
+This table matches the Theme version with the supported COmanage version.
+
+| Theme  | COmanage  |
+|:------:|:---------:|
+| v1.x.x | v3.1.x    |
+
 ## License
 
-Licensed under the Apache 2.0 license, for details see `LICENSE`.
+Licensed under the Apache 2.0 license, for details see [LICENSE](https://github.com/EGI-Foundation/comanage-registry-themeegi/blob/master/LICENSE).
```

Comment:
No need to add a link here. Please revert.

Reply:
Done